### DISCUSSION TITLE
Partition coefficient (water/container) reference to Partition coefficient (interstitial/water)

### DISCUSSION THREAD

**Discussion by wilbertdew** [Open-Systems-Pharmacology/Forum#775] [Questions & Problems] [answered]
Hi All,

I have been running into a problem a few times lately and I can't get an idea of what is going on.
In two different models in Mobi (both large molecule models, default human exported from PK-Sim) after a few additions/deletions of molecules or MSVs I get an error when I try to create a simulation that says: "Partition coefficient (water/container) with path simulation|Organism|Bone|Interstitial|mydrug|Partition coefficient (water/container) references an entity with path simulation|Neighborhoods|Bone_int_Bone_cell|mydrug|Partition coefficient (interstitial/water) that cannot be found" I get this message for all organs
I tried to find out where this comes from, and if I look at e.g. bone interstitial molecule properties I indeed see this reference (to be sure, I recreated a default large molecule human model in PK-Sim, so I have made no changes to this model).
![image](https://user-images.githubusercontent.com/8969023/90136011-cf78cf00-dd73-11ea-89d9-c35b8a9ca174.png)

When I check this reference in the Neighbourhoods, I don't see the Partition coefficient (interstitial/water)

![image](https://user-images.githubusercontent.com/8969023/90136314-46ae6300-dd74-11ea-9e7c-92391b0158f8.png)

In the default case, there is no problem of course, and I can check the simulation parameters where I see that for my specific molecule, this Partition coefficient (interstitial/water) is present and refers to the pls_int neighbourhood to another Partition coefficient (interstitial plasma) (which depends on organism parameters and the fu) and the fu.
![image](https://user-images.githubusercontent.com/8969023/90137672-3f885480-dd76-11ea-9626-94f133d8aba8.png)

From all of this, I don't get many clues but only more questions. How are these parameters generated and how does that depend on MSV/PSV? Anybody who can help me out? Much appreciated!
Best,
Wilbert

**Comment by prvmalik** [comment, marked as answer]
Here's a workaround. Save your project before you do this. The value for the K_water_int is always 1.04 for a large molecule.

Open Spatial Structure|Formulas in your building block. Sort by Name by clicking on "name." Look for the formula "PARAM_K_Water_INT_organ."
![Wilbert1](https://user-images.githubusercontent.com/29066045/90152112-7437e400-dd55-11ea-9bc2-b73012487bf8.JPG)

Press the X next to the reference to K_water_int to delete it. In the bottom panel, enter the constant "1.04" as shown. Do this for all organs.
![Wilbert2](https://user-images.githubusercontent.com/29066045/90152394-c4af4180-dd55-11ea-95a5-868ffdba0c93.JPG)

Let me know if that works.

**Comment by StephanSchaller** [comment]
Not sure if this might be due to adding/removing molecules, but what are the settings for your molecule (for which you get these errors) in the Molecule BB. Make sure a Method is selected for `DistributionInterstitial`:
![image](https://user-images.githubusercontent.com/25477643/90158846-d31a4f00-dd8f-11ea-8dfb-a92684a82546.png)

This partitioning parameter is added when the simulation is created (@Yuri05 , does it make sense for this one as there seems to be only one method that can be selected?)

**Comment by wilbertdew** [comment]
@prvmalik Thank you, that solved the problem indeed!
@StephanSchaller the calculation methods were set correctly, but I think it was because I removed the MSVs for intracellular and blood cells.

**Comment by StephanSchaller** [comment]
> but I think it was because I removed the MSVs for intracellular and blood cells.

Yes, that's likely. The parameter won't be created then, I guess, and the other parameter referencing it causes the error.

**Comment by yishuanwu** [comment]
Hi, 
I was using the above workaround at first, but I find a full solution for the same issue here:
https://github.com/Open-Systems-Pharmacology/Forum/issues/46

Hope that this could be helpful for someone else.
